distribution/scripts/jmeter/run-performance-tests.sh

```diff
@@ -121,6 +121,32 @@ declare -A test_scenario6=(
     [skip]=false
 )
 declare -A test_scenario7=(
+    [name]="h2_h2_client_downgrade"
+    [display_name]="HTTP/2 client downgrade service (h2 -> h1)"
+    [description]="An HTTP/2 (over TLS) client, which sends requests to an HTTPS back-end service and gets HTTPS responses."
```

Comment:
Shall we rephrase the description to "An HTTP/2 (with TLS) client, sends requests to an HTTP/1.1(with TLS) back-end service. With ALPN negotiation, the client connection is downgraded to HTTP/1.1(with TLS)."